docs/getting_started/howto/adaptive_localization.ipynb

```diff
@@ -0,0 +1,287 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "28648753-5876-4462-a619-a51c323e8351",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# Import libraries\n",
+    "import numpy as np\n",
+    "\n",
+    "rng = np.random.default_rng(1)\n",
+    "import seaborn as sns\n",
+    "import matplotlib.pyplot as plt"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "feae478a-b10c-41ba-b9cb-16f4c8d9c26e",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "# Adaptive Localization\n",
+    "\n",
+    "Parameter estimation using few samples (realizations) leads to sampling errors and spurious correlations (see Section 1.3), which can lead to inaccurately strong updates during the estimation process.\n",
+    "A strong update, characterized by a posterior with low variance, can be misleading - essentially leading to confident but potentially incorrect conclusions.\n",
+    "Localization is a widely used method to mitigate this.\n",
+    "\n",
+    "Localization uses only observations deemed close when updating a subset of adjacent parameters.\n",
+    "The process involves two key decisions: determining what constitutes adjacent parameters and defining a measure of closeness.\n",
+    "\n",
+    "- **Localization - Step-by-Step:**\n",
+    "    - **Select Adjacent Parameters:** Identify a subset of adjacent parameters. Subset may consist of a single parameter.\n",
+    "    - **Identify Close Observations:** Choose observations that are near this subset.\n",
+    "        Closeness can be defined in various ways.\n",
+    "    - **Update:** Use only close observations in the update step.\n",
+    "\n",
+    "Two things need to be decided.\n",
+    "What are adjacent parameters, and how to measure closeness?\n",
```

Comment:
The first sentence of this block and the last are more or less saying the same?

Reply:
Fixed